src/spyglass/common/common_behav.py

```diff
@@ -67,6 +67,9 @@ def insert_from_nwbfile(cls, nwb_file_name):
                 key["import_file_name"] = ""
                 cls.insert1(key)
 
+        # make map from epoch intervals to position intervals
+        populate_position_interval_map_session(nwb_file_name)
```

Comment:
Just making sure this doesn't run into the same error?

Reply:
It worked when I tested it out. I think the difference is this is not inside of a make function called by datajoint's autopopulate, which is the specific issue from before